phpBB/composer.json

```diff
@@ -33,7 +33,7 @@
 		"marc1706/fast-image-size": "1.1.*",
 		"paragonie/random_compat": "^1.2",
 		"patchwork/utf8": "1.1.*",
-		"s9e/text-formatter": "^0.4.2",
+		"s9e/text-formatter": "^0.5.0",
```

Comment:
Not in pre-1.0 versions, where Composer [*"acts with safety in mind and treats \^0.3 as >=0.3.0 <0.4.0."*](https://getcomposer.org/doc/articles/versions.md#caret)